pyspecProcScripts/third_level/rough_table_of_integrals.py

```diff
@@ -162,7 +166,9 @@ def rough_table_of_integrals(
     )
     # }}}
     s = s[direct:signal_range].real.integrate(direct).set_error(None)
-    psd.plot(s, "-o", ax=ax4)
+    if inc_plot_color:
+        s.set_plot_color_next()
+    psd.plot(s, "o", ax=ax4)
```

Comment:
there is a plot color property of nddata that you can get and set with `set_plot_color` and `get_plot_color`.  Any copies/etc of your nddata then inherit that property, making it easy to color a single dataset in the same way.  This function here sets the property based on the default matplotlib color cycle.